### PR TITLE
Update AJAX Request property descriptions to note that all AJAX Reque…

### PR DESCRIPTION
**Pull request by metal-messiah**
### What problems does this PR solve?
* Browser agent is releasing an update which allows all AJAX requests to be captured.  This updates the docs to account for that.

### Are you making a change to site code?
No

### CODE REVIEW

**Comment by urbiz-nr** [comment]
Thanks @metal-messiah! Get in touch with the hero in #documentation whenever you need a review!

**Comment by barbnewrelic** [comment]
I am wondering if this is related to https://github.com/newrelic/docs-website/issues/2974 ? I've filed a Jira for that one and added you as a watcher, just in case.

File: src/content/docs/telemetry-data-platform/understand-data/event-data/events-reported-browser-monitoring.mdx [hunk 1]
```diff
@@ -73,11 +73,11 @@ redirects:
 
     <tr>
       <td>
-        [`AjaxRequest` (SPA](/attribute-dictionary/?attribute_name=&events_tids%5B%5D=8302&event=AjaxRequest))
+        [`AjaxRequest`](/attribute-dictionary/?attribute_name=&events_tids%5B%5D=8302&event=AjaxRequest)
       </td>
 
       <td>
-        An `AjaxRequest` event is created automatically when an Ajax request occurs during a `BrowserInteraction` event. The event attributes track geographic and browser info.
+        An `AjaxRequest` event is created any time an Ajax call is made. This can happen either automatically when an Ajax request occurs during a `BrowserInteraction` event or during any stand-alone Ajax request. The event attributes track geographic and browser info.
```
Comment (martinkuba) [inline src/content/docs/telemetry-data-platform/understand-data/event-data/events-reported-browser-monitoring.mdx:80]: I would take out the second sentence and instead say something like -
"When an ajax call happens during a BrowserInteraction event, it is decorated with additional attributes related to the browser interaction"

File: src/data-dictionary/events/AjaxRequest/timeToLastCallbackEnd.md [hunk 2]
```diff
@@ -4,7 +4,8 @@ type: attribute
 units: seconds (s)
 events:
   - AjaxRequest
+  - BrowserInteraction
   - BrowserTiming
 ---
 
-The duration, in seconds, from the start of the request (timestamp) to the end of the last callback. This is not just an additive function; the callback time can overlap with the wait time.
\ No newline at end of file
+The duration, in seconds, from the start of the request (timestamp) to the end of the last callback. This is not just an additive function; the callback time can overlap with the wait time. This attribute is only seen in AjaxRequest Events that are tied to BrowserInteraction Events.
```
Comment (martinkuba) [inline src/data-dictionary/events/AjaxRequest/timeToLastCallbackEnd.md:11]: The same applies to these attributes:
browserInteractionId, browserInteractionName, eventId, parentEventId, jsDuration, timeSinceBrowserInteractionStart, timeToSettle

I suppose the only ones that are not self-explanatory are jsDuration and timeToSettle. So, I would maybe add the same sentence to these two.

**Comment by martinkuba** [comment]
I think we will also need a new page on the ajax deny list configuration. We have a [Configuration section](https://docs.newrelic.com/docs/browser/new-relic-browser/configuration/) in the docs, so maybe it should go there.
@barbnewrelic @rr0214  Do you have a preference where a new configuration section (in the Browser App Settings page) should go?

**Comment by bradleycamacho** [comment]
@martinkuba @barbnewrelic This has gotten quite stale. Any updates on the progress of this PR?

**Comment by barbnewrelic** [comment]
I'm following up with @martinkuba separately about his Ajax deny list question. There's already some info about allow lists and deny lists in https://docs.newrelic.com/docs/browser/new-relic-browser/configuration/group-browser-metrics-urls/  , including Ajax. I don't think this needs a separate doc but will confirm. In the meantime I'll finish up your other recommended changes. Thanks for reviewing this info with a broader view of what additional documentation may be needed!

**Review by barbnewrelic — APPROVED** [review]
I made some additional edits to metal-messiah's original changes based on feedback from martinkuba. Also fixed the merge conflict.

**Pull request merged**